src/main/io/display.c

```diff
@@ -573,6 +571,11 @@ void updateDisplay(void)
                 (((int32_t)(now - pageState.nextPageAt) >= 0L && (pageState.pageFlags & PAGE_STATE_FLAG_CYCLE_ENABLED)));
         if (pageState.pageChanging && (pageState.pageFlags & PAGE_STATE_FLAG_CYCLE_ENABLED)) {
             pageState.cycleIndex++;
+            
+            if (cyclePageIds[pageState.cycleIndex] == PAGE_BATTERY && !feature(FEATURE_VBAT) && !feature(FEATURE_CURRENT_METER)) {
```

Comment:
prefer `... !(feature(FEATURE_VBAT) || feature(FEATURE_CURRENT_METER))` here.  the grouping is meaningful and the less `!` operators the better.